data/json/items/gunmod/stock.json

```
@@ -431,5 +507,139 @@
         "max_contains_weight": "200 g"
       }
     ]
+  },
+  {
+    "id": "shotshell_stock",
+    "type": "GUNMOD",
+    "name": { "str": "speedload shotgun stock" },
+    "description": "A fixed stock with two spring-loaded tubes, each holding two shotshells.",
+    "weight": "350 g",
+    "volume": "500 ml",
+    "integral_volume": "0 ml",
+    "integral_weight": "0 g",
+    "price": "400 USD",
+    "price_postapoc": "5 USD",
+    "install_time": "10 m",
+    "to_hit": 1,
+    "material": [ "plastic" ],
+    "symbol": ":",
+    "color": "dark_gray",
+    "location": "stock",
+    "mod_targets": [ "shotgun" ],
+    "melee_damage": { "bash": 6 },
+    "pocket_mods": [
+      {
+        "description": "Shotshell tube.",
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "ammo_restriction": { "20x66mm": 2, "410shot": 2, "shot": 2 }
+      },
+      {
+        "description": "Shotshell tube.",
+        "pocket_type": "CONTAINER",
+        "rigid": true,
+        "ammo_restriction": { "20x66mm": 2, "410shot": 2, "shot": 2 }
+      }
+    ],
+    "min_skills": [ [ "weapon", 2 ] ]
+  },
+  {
+    "id": "stock_ammo_loops_canvas",
+    "type": "GUNMOD",
+    "name": { "str": "buttstock ammunition loops" },
+    "description": "Cloth loops on the shoulder stock of a long arm holding five to 10 individual shotshells or rifle cartridges.",
```

```suggestion
    "description": "Cloth loops on the shoulder stock of a long arm holding five to ten individual shotshells or rifle cartridges.",
```
Shouldn't swap between words & numbers in the same phrase